src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import "./Todo.css"
+class TodoForm extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        todo: ''
+      };
+    }
```

Comment:
having this nested component hold its own state instead of passing down more methods and state properties as props from App.js does make writing the code in here a little easier to understand, just make sure you understand how it would still work if this component was a stateless, functional component and App was controlling everything and passing all state down